README.md

```diff
@@ -41,7 +57,8 @@ Breaking changes in the protocol are indicated by incrementing the minimum reade
 
 ## Roadmap
```

Comment:
Under the Data Storage Compatibility section, please update the protocol action link to `https://github.com/delta-io/delta/blob/master/core/src/test/scala/org/apache/spark/sql/delta/ActionSerializerSuite.scala`